PythonTool/WindowSettings.py

```diff
@@ -0,0 +1,52 @@
+import os
+
+from visit import *
+
+
+def WindowSettings(Shading=False):
```

Comment:
When you say 'place each setting as an argument' you mean add arguments to the WindowSettings function?